src/index.js

```diff
@@ -29,8 +29,10 @@ class Toggle extends Component {
   getTogglerProps = (props = {}) => ({
     'aria-controls': 'target',
     'aria-expanded': Boolean(this.getOn()),
+    tabIndex: props.tabIndex == null ? 0 : props.tabIndex,
```

Comment:
Could probably just do: `tabIndex: 0` and the `...props` will override `tabIndex` if it's on there.

Reply:
Duly noted. I'll update this later today.